src/web/mjs/connectors/templates/WordPressMadara.mjs

```diff
@@ -56,23 +56,47 @@ export default class WordPressMadara extends Connector {
         });
     }
 
+    async _getChaptersAjaxOld(dataID) {
+        let uri = new URL(this.path + '/wp-admin/admin-ajax.php', this.url);
+        let request = new Request(uri, {
+            method: 'POST',
+            body: 'action=manga_get_chapters&manga=' + dataID,
+            headers: {
+                'content-type': 'application/x-www-form-urlencoded',
+                'x-referer': this.url
+            }
+        });
+        const data = await this.fetchDOM(request, this.queryChapters);
+        if(data.length) {
+            return data;
+        } else {
+            throw new Error('No chapters found (old ajax endpoint)!');
+        }
+    }
+
+    async _getChaptersAjaxNew(mangaID) {
```

Comment:
I think that you can omit the "new" in the function name, but let old in the "old one"

Reply:
Done